BossMod/BossModule/ArenaBounds.cs

```diff
@@ -11,6 +10,8 @@ public abstract record class ArenaBounds(float Radius, float MapResolution)
     public RelSimplifiedComplexPolygon ShapeSimplified { get; private set; } = new();
     public List<RelTriangle> ShapeTriangulation { get; private set; } = [];
     private readonly PolygonClipper.Operand _clipOperand = new();
+    public static readonly Dictionary<object, object> StaticCache = [];
+    public readonly Dictionary<object, object> Cache = [];
```

Comment:
Not this again... I've already seen this proposed a few times and rejected that - there is a cache on a different level (miniarena), I don't see a good reason to introduce yet another cache here. If it gives some performance benefits, I will need to understand why exactly it can't be done on the same level...